packages/website/src/css/custom.css

```diff
@@ -191,3 +191,25 @@ h6 {
 [data-theme='dark'] .code-block-added-line {
   background-color: #00ff9510;
 }
+
+a {
+  text-decoration: underline;
+}
+
+/* Removes underlines from table of contents, next page button,
+menu (left hand sidebar), edit this page button,
+and footer links (github, twitter, etc), buttons,
+navbar links, the navbar title, blog sidebar, and blog post headers
+respectfully */
+a.table-of-contents__link,
+a.pagination-nav__link,
+a.menu__link,
+a.theme-edit-this-page,
+footer *,
+.button,
+.navbar__link,
+.navbar__brand,
+aside *,
+header * {
+  text-decoration: none;
+}
```

Comment:
[Request] I think this approach works well for the how the site is structured now, but:

* `*`s are kind of hard to work with long-term. If we add in more cascading layers of styles they'll be hard to override.
* Similarly, this is a list based on the structure & stuff we use now. But if that changes this list will have to change.

If I understand it right, this approach is making all anchors underlined by default, then disabling that for certain others. How about going the opposite direction instead - targeting only the anchors we know we want to be underlined? I think that's doable by selecting all anchors inside markdown, like `.markdown a`. Or maybe even only the anchors inside paragraphs. Thoughts?

Reply:
I agree, thank you for the idea, that should work perfectly.